ee-feature-pack/common/src/main/resources/modules/system/layers/base/org/jboss/ws/cxf/jbossws-cxf-server/main/module.xml

```diff
@@ -69,6 +70,8 @@
         </module>
         <module name="org.jboss.logging" />
         <module name="org.apache.ws.security" />
+        <module name="org.picketbox" />
```

Comment:
Is this added back by mistake?

Reply:
@bstansberry Yes that was a mistake, fixed now. Thank you